packages/plugins/GoPlusSecurity/src/SNSAdaptor/components/SearchBox.tsx

```diff
@@ -0,0 +1,126 @@
+import { Box, Button, InputAdornment, MenuItem, Stack, Typography } from '@mui/material'
+import { useAsync } from 'react-use'
+import { memo, useMemo, useState } from 'react'
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown'
+import { makeStyles, MaskColorVar, MaskTextField, ShadowRootMenu } from '@masknet/theme'
+import { SearchIcon } from '@masknet/icons'
+import { useI18N } from '../../locales'
+import type { SecurityAPI } from '@masknet/web3-providers'
+import { GoPlusLabs } from '@masknet/web3-providers'
+import { ChainId, getChainDetailed } from '@masknet/web3-shared-evm'
+import parseInt from 'lodash-es/parseInt'
+
+const useStyles = makeStyles()((theme) => ({
+    root: {
+        width: '600px',
+    },
+    content: {
+        width: '552px',
+    },
+    selectedButton: {
+        width: '100%',
+        height: '100%',
+        fontWeight: 400,
+        borderColor: theme.palette.divider,
+    },
+    searchButton: {
+        borderRadius: 8,
+    },
+    menu: {},
+    search: {
+        backgroundColor: 'transparent !important',
+        border: `solid 1px ${MaskColorVar.twitterBorderLine}`,
+        height: 48,
+        borderColor: theme.palette.divider,
+        borderRadius: 12,
+    },
+}))
+
+interface SearchBoxProps {
+    onSearch(chainId: string, content: string): void
+}
+
+const DEFAULT_SEARCH_CHAIN = '1'
```

Comment:
```suggestion
const DEFAULT_SEARCH_CHAIN = ChainId.Mainnet
```

Reply:
I converted the chain id type in response from string to ChainId. https://github.com/DimensionDev/Maskbook/pull/5873/commits/6ca4e68f7c12d1008b274ee1f981ef465c780c7b